build/readme.loader.cts

```diff
@@ -0,0 +1,8 @@
+// Can't use modules because this is a bizarropackscript world
+module.exports = async function ReadmeLoader () {
```

Comment:
In reality we get the readme’s source here already, so it could use it instead of reading it from disk again